FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/V2DemoListActivity.kt

```diff
@@ -520,6 +524,12 @@ class V2DemoListActivity : ComponentActivity() {
                                     trailingAccessoryContent = if (it.badge != Badge.None) {
                                         {
                                             Badge(
+                                                modifier = when(it.badge) {
```

Comment:
Instead of applying modifier, create Tokens for each type & pass it based on Enum value

Reply:
Done